### PR TITLE
feat(starfish): various ui improvements to page overview and slideout panels

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
minor ui improvements and cleanup

### CODE REVIEW

File: static/app/views/performance/browser/webVitals/pageOverview.tsx [hunk 1]
```diff
@@ -93,6 +95,17 @@ export default function PageOverview() {
     return null;
   }
 
+  const transactionSummaryTarget =
+    project &&
+    !Array.isArray(location.query.project) && // Only navigate to transaction summary when one project is selected.
```
Comment (AbhiPrasad) [inline static/app/views/performance/browser/webVitals/pageOverview.tsx:100]: so the button doesn't do anything if no projects are selected? Should conditionally display a link if so?

**Review by AbhiPrasad — APPROVED** [review]

File: static/app/views/performance/browser/webVitals/pageOverview.tsx (hunk 1 above)
Comment (edwardgou-sentry) [reply static/app/views/performance/browser/webVitals/pageOverview.tsx:100]: The comment could use an update though, I think thats where the confusion is.

Comment (Abdkhan14) [reply static/app/views/performance/browser/webVitals/pageOverview.tsx:100]: It doesn't get rendered at all for multiple projects

**Review by Abdkhan14 — APPROVED** [review]
Lgtm, like the smaller performance score ring. Tested things out locally. 🚢

**Pull request merged**